avro-codegen/src/test/java/com/linkedin/avroutil1/codegen/SpecificRecordClassGeneratorTest.java

```diff
@@ -22,176 +21,43 @@
 
 
 public class SpecificRecordClassGeneratorTest {
-
-  @Test
```

Comment:
what happened to all these tests?

Reply:
All still exist just they now share a parameterized implementation after collapsing redundant driver code. Originated from trying to understand and enhance coverage before moving on due to existing generated object file compilation issues. Left as a craftsmanship/readability improvement but happy to drop if preferred.

Reply:
Point taken. My bad, was lazy after fighting recent environment changes.